specs/latest/1.0/index.html

```diff
@@ -395,6 +395,38 @@ <h3><a name="THE_DRAWING_BUFFER">The Drawing Buffer</a></h3>
         </p>
     </div>
 
+    <dl class="methods">
+        <dt class="idl-code">undefined backbufferStorage(GLenum sizedFormat, unsigned long width, unsigned long height);
+        <dd>
+            Respecify the size and format of the backbuffer,
+            similar to <code>renderbufferStorage</code> as used on Renderbuffers.
+
+            Clearing behavior is equivalent to setting <code>HTMLCanvasElement.width</code> and <code>HTMLCanvasElement.height</code>, followed by changing the backbuffer format.
+            This method respects <code>WebGLContextAttributes.antialias</code>.
```

Comment:
How can the user know (aside from calling gl.getError, which is discouraged) that the setting actually took effect? Should we add a new read-only property alongside `drawingBufferWidth` and `drawingBufferHeight`?

Reply:
You can't tell today for textures or renderbuffers, so I'm not super worried about going above-and-beyond for backbuffers.